scripts/test/Muon/grouping_tab/calculate_pair_and_group_test.py

```diff
@@ -7,7 +7,7 @@
 import unittest
 from unittest import mock
 from Muon.GUI.Common.calculate_pair_and_group import _get_MuonGroupingCounts_parameters, \
-    _get_MuonGroupingAsymmetry_parameters, _get_MuonPairingAsymmetry_parameters
+    _get_MuonPairingAsymmetry_parameters, _get_EstimateMuonAsymmetryFromCounts_parameters
```

Comment:
PEP8 convention is methods with prefixed `_` are private and shouldn't be called
Should these be renamed to `get_MuonParingAsymmetry_parameters` to reflect the fact they are called outside the original file now?

Reply:
Since this is just in the test file, do these need to be renamed? As they are only called explicitly for testing purposes not in the main code base

Reply:
Oh my bad, I misread which file it was in
For the purposes of testing / mocking this is fine :+1: